src/TestEnvironment.Docker.Containers.Postgres/PostgresContainerWaiter.cs

```diff
@@ -6,39 +6,33 @@
 
 namespace TestEnvironment.Docker.Containers.Postgres
 {
-    public class PostgresContainerWaiter : IContainerWaiter<PostgresContainer>
+    public class PostgresContainerWaiter : BaseContainerWaiter<PostgresContainer>
     {
-        private readonly ILogger _logger;
-
         public PostgresContainerWaiter(ILogger logger)
+            : base(logger)
         {
-            _logger = logger;
         }
 
-        public async Task<bool> Wait(PostgresContainer container, CancellationToken cancellationToken)
+        protected override async Task<bool> PerformCheckAsync(PostgresContainer container,
```

Comment:
Make it single-line or move this parameter to the new line

Reply:
Fixed